### ISSUE TITLE
Changelog has no dates

### ISSUE THREAD

**Issue by alex-rx2** [MaFi-Games/Captain-of-Industry-issues#1605]
### Issue description

Ingame changelog has no dates for versions, hard to parse what's actually new is there.

### Reproduction steps

Start game after new versions was installed?
Changelog window will be shows with tons of text without any dates or other marks of what is new in this wall of text.

### Game version and build number

v0.6.4d b343

### Operating system

Windows

**Comment by poltime** [comment]
Captain Evidence suggests that the latest version on Steam is the newest.
